content/03.virology.md

```diff
@@ -1,8 +1,9 @@
 ## Virology
 
-Examining the structure and basic functions of the SARS-CoV-2 virus is useful for understanding and predicting its pathogenesis and transmission.
-Genomic analysis of the virus provided early information about its phylogenetic position among other coronaviruses.
-These viruses are widely studied and their genomes are highly conserved, leading to strong hypotheses early on about how the structure and functions of the virus. 
+Examining the structure of the SARS-CoV-2 virus and the mechanisms that allow it to enter cells and replicate can provide useful information for predicting pathogenesis and transmission.
+In the case of SARS-CoV-2, early information about the virus was identified by sequencing viral sequences found in patient BALF and identifying the virus' phylogenetic position.
+The genomes of coronaviruses are highly conserved, leading to strong hypotheses early in SARS-CoV-2 research.
+While only a limited number of coronaviruses are known to infect humans, both SARS and MERS have been studied widely. 
```

Comment:
```suggestion
While only a limited number of coronaviruses are known to infect humans, both SARS and MERS have been studied widely. 
```

Is this line a little redundant as you deal with the same topic above?

Reply:
Did you spell out BALF before?  I don't know what it means.

Reply:
Hmm it is defined in the intro, but I definitely agree it will be hard to keep track of all the acronyms in this paper! I'm not quite sure how to address it, I'll keep an eye on this as I go through the submission process in case it's OK to include a list of acronyms, which some journals allow. (In this case, it's bronchoalveolar lavage fluid!)